media-video/vdr/vdr-2.2.0-r4.ebuild

```diff
@@ -0,0 +1,304 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit flag-o-matic toolchain-funcs
+
+# Switches supported by extensions-patch
+EXT_PATCH_FLAGS="alternatechannel graphtft naludump permashift pinplugin
+				mainmenuhooks menuorg menuselection resumereset ttxtsubs"
+
+# names of the use-flags
+EXT_PATCH_FLAGS_RENAMED=""
+
+# names ext-patch uses internally, here only used for maintainer checks
+EXT_PATCH_FLAGS_RENAMED_EXT_NAME="bidi no_kbd sdnotify"
+
+IUSE="bidi debug +kbd html systemd vanilla ${EXT_PATCH_FLAGS} ${EXT_PATCH_FLAGS_RENAMED}"
+
+MY_PV="${PV%_p*}"
+MY_P="${PN}-${MY_PV}"
+S="${WORKDIR}/${MY_P}"
+
+EXT_P="extpng-${P}-gentoo-edition-v1"
+
+DESCRIPTION="Video Disk Recorder - turns a pc into a powerful set top box for DVB"
+HOMEPAGE="http://www.tvdr.de/"
+SRC_URI="ftp://ftp.tvdr.de/vdr/${MY_P}.tar.bz2
+	mirror://gentoo/${EXT_P}.patch.bz2"
+
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~x86"
+SLOT="0"
+LICENSE="GPL-2"
+
+COMMON_DEPEND="virtual/jpeg:*
+	sys-libs/libcap
+	>=media-libs/fontconfig-2.4.2
+	>=media-libs/freetype-2"
+
+DEPEND="${COMMON_DEPEND}
+	>=virtual/linuxtv-dvb-headers-5.3
+	sys-devel/gettext"
+
+RDEPEND="${COMMON_DEPEND}
+	dev-lang/perl
+	>=media-tv/gentoo-vdr-scripts-2.7
+	media-fonts/corefonts
+	bidi? ( dev-libs/fribidi )
+	systemd? ( sys-apps/systemd )"
+
+CONF_DIR=/etc/vdr
+CAP_FILE=${S}/capabilities.sh
+CAPS="# Capabilities of the vdr-executable for use by startscript etc."
+
+pkg_setup() {
+
+	use debug && append-flags -g
+
+	PLUGIN_LIBDIR="/usr/$(get_libdir)/vdr/plugins"
+
+	tc-export CC CXX AR
+}
+
+add_cap() {
+	local arg
+	for arg; do
+		CAPS="${CAPS}\n${arg}=1"
+	done
+}
+
+enable_patch() {
+	local arg ARG_UPPER
+	for arg; do
+		ARG_UPPER=$(echo $arg|tr '[:lower:]' '[:upper:]')
+		echo "${ARG_UPPER} = 1" >> Make.config
+
+		# codesnippet to bring the extpng defines into the vdr.pc CXXFLAGS CFLAGS
+		echo "-DUSE_${ARG_UPPER}" >> "${T}"/defines.tmp
+		cat "${T}"/defines.tmp | tr \\\012 ' '  > "${T}"/defines.IUSE
```

Comment:
both of these commands are missing `|| die`

Reply:
"die" added